flox-bash/tests/test_support.bash

```diff
@@ -16,7 +16,6 @@ setup_file() {
     export FLOX_CLI=$FLOX_PACKAGE/bin/flox
     export FLOX_PACKAGE_FIRST8=$(echo $FLOX_PACKAGE | dd bs=c skip=11 count=8 2>/dev/null)
   fi
-  export FLOX_DISABLE_METRICS="true"
```

Comment:
Yes I was on a quest to remove all mention of metrics, and these tests (when run from the flox-bash subdirectory) don't invoke the rust CLI so if I understand correctly cannot submit metrics. I don't believe this `test_support.bash` is used to test the rust CLI so that's why I thought it was safe to remove.

Reply:
it is imported by both bats tests which are run with the rust flox if run through cargo.
if we dont have this here we should at least add that variable when running the tests with rust.

Reply:
> we should at least add that variable when running the tests with rust.

Definitely agree, and we should take this opportunity to add it on the rust side so that we don't miss it when we remove all of flox-bash.

Reply:
Pushed commit 6e3fcd80e89bcb51d9d7a0b4d1e989487a27c5b6 to add it on the rust side - resolving conversation.  :+1: